MOKApp/app/src/main/java/mok/it/app/mokapp/fragments/RewardsFragment.kt

```diff
@@ -89,7 +89,7 @@ class RewardsFragment : Fragment() {
                     loadImage(ivImg, model.icon)
                     holder.itemView.rewardName.text = model.name
                     holder.itemView.rewardPrice.text = model.price.toString()
-                    if (userModel.badges >= model.price) {
+                    if (userModel.projectBadges.values.sum() >= model.price) {
```

Comment:
Itt is ki kellene vonni a már elköltött mancsokat, különben akárhányszor request-elheti a user a jutalmakat, nem fog fogyni a mancsa.